infrastructure/cli/environment_test.go

```diff
@@ -41,11 +45,22 @@ func TestAddConfigValuesToEnv(t *testing.T) {
 		c.SetIntegrationVersion(expectedIntegrationVersion)
 		c.SetIdeVersion(expectedIdeVersion)
 		c.SetIdeName(expectedIdeName)
+		s := oauth2.Token{
```

Comment:
Should we keep the old test and just add a new oauth2 env test?

Reply:
Hm, the expected default behaviour changes, so I would change this test, too.

Reply:
I can add a test for the Token auth, though :)